eclipse-scout-core/src/desktop/outline/Outline.ts

```diff
@@ -159,14 +161,36 @@ export class Outline extends Tree implements DisplayParent, OutlineModel {
     this._nodesSelectedInternal(this.selectedNodes);
   }
 
-  /**
-   * This function returns the outline mediator instance. When we're in an online Scout application we must
-   * return a null instance here, because mediation is done server-side.
-   */
   protected _createMediator(): OutlineMediator {
     return scout.create(OutlineMediator);
   }
 
+  protected _installLocalTreeListener() {
+    // Note: Although there is a Java outline mediator in Scout Classic, we always install the JS outline mediator.
+    // This enables hybrid outlines, i.e. Java outlines that contain both Java and JS pages.
+
+    // Mediate node selection
+    this.on('nodesSelected', (event: TreeNodesSelectedEvent) => {
+      this.mediator.onPageSelected(this.selectedNode());
+    });
+
+    // Mediate subtree changes
+    let handleSubTreeChanged = (nodes: TreeNode | TreeNode[]) => {
```

Comment:
Can you explain in the commit message, what the purpose of this change is? Is this relevant for the navigate down button?

Reply:
A more detailed description was added.